Packs/FeedZoom/Integrations/FeedZoom/FeedZoom.py

```diff
@@ -0,0 +1,191 @@
+        """
+        super().__init__(base_url, verify=verify, proxy=proxy)
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        result = []
+        r = self._http_request('GET', url_suffix='', full_url=self._base_url, resp_type='text')
+
+        soup = BeautifulSoup(r, 'html.parser')
+
+        try:
+            raw_data: List = sum([cell.get_text(strip=True, separator=" ").split(" ")
+                                 for cell in soup.select(".pure-table tbody tr td")], [])
+            cidrs = list(set(raw_data))
+            for cidr in cidrs:
+                if re.match(ipv4cidrRegex, cidr):
+                    result.append({
+                        "value": cidr,
+                        'type': FeedIndicatorType.CIDR,
+                        "FeedURL": self._base_url
+                    })
+
+        except requests.exceptions.SSLError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
+                            f'Check your not secure parameter.\n\n{err}')
+        except requests.ConnectionError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
+                            f'Check your Server URL parameter.\n\n{err}')
+        except requests.exceptions.HTTPError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n')
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data to Json. \n\nError massage: {err}')
+
+        return result
+
+
+def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], Dict[Any, Any]]:
+    """Builds the iterator to check that the feed is accessible.
+    Args:
+        client: Client object.
+    Returns:
+        Outputs.
+    """
+    client.build_iterator()
+    return 'ok', {}, {}
+
+
+def fetch_indicators(client: Client, feed_tags: List = [], limit: int = -1) -> List[Dict]:
+    """Retrieves indicators from the feed
+    Args:
+        client (Client): Client object with request
+        feed_tags (list): tags to assign fetched indicators
+        limit (int): limit the results
+    Returns:
+        Indicators.
+    """
+    iterator = client.build_iterator()
+    indicators = []
+    if limit > 0:
+        iterator = iterator[:limit]
+    for item in iterator:
+        value = item.get('value')
+        type_ = item.get('type', FeedIndicatorType.Domain)
+        raw_data = {
+            'value': value,
+            'type': type_,
+        }
+        for key, val in item.items():
+            raw_data.update({key: val})
+        indicator_obj = {
```

Comment:
Please add the following key to the `indicator_obj`, so it would be easier to identify the source of the indicator:
`'fields': {'service': 'Zoom Feed'}`